sbin/post-commit-bg

```diff
@@ -100,14 +100,22 @@ main() {
     trac_hook "$REPOS" "$REV" added || RET_CODE=$?
 
     # Check size - send warning email if threshold exceeded
-    local REV_FILE=$REPOS/db/revs/$((REV / 1000))/$REV
-    local REV_FILE_SIZE_THRESHOLD=1048576 # 1MB
-    local REV_FILE_SIZE=$(du -b -s $REV_FILE | cut -f 1)
-    if (($REV_FILE_SIZE > $REV_FILE_SIZE_THRESHOLD)); then
-        echo "REV_FILE_SIZE=$REV_FILE_SIZE # EXCEED $REV_FILE_SIZE_THRESHOLD"
+    local MB=1048576
+    local THRESHOLD=10
+    local SIZE_THRESHOLD_FILE="${REPOS}/hooks/pre-commit-size-threshold.conf"
+    if [[ -f "${SIZE_THRESHOLD_FILE}" && -r "${SIZE_THRESHOLD_FILE}" ]]; then
+        THRESHOLD=$(<"${SIZE_THRESHOLD_FILE}")
+    fi
+    local REV_FILE="${REPOS}/db/revs/$((${REV} / 1000))/${REV}"
+    local REV_FILE_SIZE=$(du -b -s "${REV_FILE}" | cut -f 1)
```

Comment:
It is not a change here, but I think it was to ensure that we get a single number for size whether `REV_FILE` is a file or directory.